pages/windows/mkdir.md

```diff
@@ -0,0 +1,11 @@
+# mkdir
+
+> Creates a directory.
+
+- Create a directory:
+
+`mkdir {{directory_name}}`
+
+- Recursively create a nested direcotry tree:
+
+`mkdir {{path/to/sub_directory_name}}`
```

Comment:
There seems to be no difference between the first and second command. Is this correct ?

Reply:
Yep. I wanted to show explicitly that the windows variant of `mkdir` recursively creates directories by default, whereas the Linux one doesn't